src/pages/About/utils/Functions.ts

```diff
@@ -0,0 +1,9 @@
+export const handleHover = (str:string) => {
+    const div = document.getElementById(str)
+    div && div?.setAttribute("style", "color: white")
+  }
+  
+export const handleHoverEnd = (str:string) => {
+    const div = document.getElementById(str)
+    div && div?.setAttribute("style", "color: #bbbabc")
```

Comment:
Can you somehow use refs instead of getElementById and setAttribut?


Why?
"In React, it is generally not recommended to use getElementById to manipulate the DOM directly. This is because React operates using a virtual DOM, which is a lightweight copy of the actual DOM.

React components are responsible for rendering the virtual DOM, and React itself handles updating the actual DOM when necessary. If you use getElementById to directly manipulate the actual DOM, you risk causing conflicts and inconsistencies between the virtual and actual DOMs, which can lead to bugs and unexpected behavior."

Reply:
I ended up changing this by recreating the four lines I am targeting with getElementByID - mapping a state Array of list items, and using the onHoverStart/onHoverEnd function to update the Array. Then changing the style in css.

I'm happy to redo this with useRef. I got so caught up in solving the problem I forgot you'd said that